Pythonwin/pywin/framework/mdi_pychecker.py

```diff
@@ -362,8 +362,8 @@ def threadPycheckerRun(self):
                 result = "Can't find python.exe!\n"
             elif not os.path.isfile(pychecker):
                 result = (
-                    "Can't find checker.py - please install pychecker "
-                    "(or run 'setup.py install' if you have the source version)\n"
+                    "Can't find checker.py - please install PyChecker "
+                    "(https://pypi.org/project/PyChecker/)\n"
```

Comment:
This code is getting removed by https://github.com/mhammond/pywin32/pull/2412 anyway